functions.php

```diff
@@ -24,8 +24,11 @@ function common_design_theme_stylesheets() {
 	wp_enqueue_style ('common-design-normalize', get_template_directory_uri().'/resources/assets/css/normalize.css', array(), null, 'all' );
 	wp_enqueue_style ('common-design-hidden', get_template_directory_uri().'/resources/assets/css/hidden.module.css', array(), null, 'all' );
 	wp_enqueue_style ('common-design-core', get_template_directory_uri().'/resources/assets/css/cd.css', array(), null, 'all' );
-	wp_enqueue_style ('common-design-button', get_template_directory_uri().'/resources/assets/css/cd-button.css', array(), null, 'all' );
-	wp_enqueue_style ('common-design-page-title', get_template_directory_uri().'/resources/assets/css/cd-page-title.css', array(), null, 'all' );
+	wp_enqueue_style ('common-design-flow', get_template_directory_uri().'/resources/assets/css/cd-flow/cd-flow.css', array(), null, 'all' );
```

Comment:
@anna-prince I noticed the CSS files were not added to the functions file, maybe I forgot to commit the changes I had last week. I'm adding them back in this PR and notice I've also added the sub directory for each component so their readme is included.